mmit/core/piecewise_function.cpp

```diff
@@ -165,7 +165,7 @@ int PiecewiseFunction::insert_point(double y, bool is_upper_bound) {
     }
 
     // Breakpoint info
-    float b = y - s * this->margin;
```

Comment:
although I would not expect that it would make a big difference in terms of test error...